python/inputmodule/gui/ledmatrix.py

```diff
@@ -37,15 +41,17 @@ def countdown(dev, seconds):
 
         time.sleep(0.01)
 
-    light_leds(dev, 306)
-    breathing(dev)
-    # blinking(dev)
+    if get_status() == 'countdown':
+        light_leds(dev, 306)
+        breathing(dev)
+        # blinking(dev)
 
 
 def blinking(dev):
     """Blink brightness high/off every second.
     Keeps currently displayed grid"""
-    while True:
+    set_status('blinking')
+    while True and get_status() == 'blinking':
```

Comment:
Nitpick: `True and` is redundant boolean logic
```suggestion
    while get_status() == 'blinking':
```